o4/o4.py

```diff
@@ -917,6 +951,11 @@ def man(cmd):
         print(f'*** INFO: {os.getcwd()} is already synced to {actual_cl}, use -f to force a'
               f' full verification.')
 
+    if prep:
+        err_print("*** INFO: Sync from p4 is complete. Hybrid import upstream changes to git.")
```

Comment:
"Sync from p4 is complete. Hybrid import of upstream changes to git will commence."